### PR TITLE
METRON-140 handle empty line in CSVExtractor

### PR DESCRIPTION
**Pull request by mmiklavc**
https://issues.apache.org/jira/browse/METRON-140

### CODE REVIEW

File: metron-platform/metron-data-management/src/main/java/org/apache/metron/dataloads/extractor/csv/CSVExtractor.java [hunk 1]
```diff
@@ -82,6 +83,14 @@ public LookupConverter getConverter() {
     return Arrays.asList(new LookupKV(key, converter.toValue(values)));
   }
 
+  private boolean ignore(String line) {
+    if(null == line) {
+      return false;
```
Comment (cestella) [inline metron-platform/metron-data-management/src/main/java/org/apache/metron/dataloads/extractor/csv/CSVExtractor.java]: I think you mean to return "true" here

File: metron-platform/metron-data-management/src/test/java/org/apache/metron/dataloads/extractor/csv/CSVExtractorTest.java [hunk 2]
```diff
@@ -91,5 +85,9 @@ public void validate(ExtractorHandler handler) throws IOException {
       Iterable<LookupKV> results = handler.getExtractor().extract("#google.com,1.0,foo");
       Assert.assertEquals(0, Iterables.size(results));
     }
+    {
+      Iterable<LookupKV> results = handler.getExtractor().extract("");
+      Assert.assertEquals(0, Iterables.size(results));
+    }
```
Comment (cestella) [inline metron-platform/metron-data-management/src/test/java/org/apache/metron/dataloads/extractor/csv/CSVExtractorTest.java]: Cool!  Can you add a case for null and for a "  "?

**Comment by cestella** [comment]
This looks great outside of the couple of things I mentioned.  Definitely like the null check.  +1 imminent if you get those changes.

**Comment by mmiklavc** [comment]
I also tested this on single-node Vagrant

**Comment by cestella** [comment]
+1

**Comment by dlyle65535** [comment]
+1

**Pull request closed without merging**